tensorflow_datasets/structured/netflix_shows/netflix_shows.py

```diff
@@ -0,0 +1,127 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""netflix_shows dataset."""
+
+import collections
+import csv
+import numpy as np
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets.public_api as tfds
+
+_DESCRIPTION = """
+This dataset consists of TV shows and movies available on Netflix.
+The dataset is collected from Flixable which is a third-party Netflix search engine.
+Float and int missing values are replaced with -1,
+string missing values are replaced with 'Unknown'.
+"""
+
+_CITATION = """
+"""
+
+_URL = "https://rushabh-v.github.io/datasets/netflix_shows.csv"
+
+RATING_CLASSES = [
+    'Unknown', 'G', 'NC-17', 'NR', 'PG', 'PG-13', 'R', 'TV-14', 'TV-G', 'TV-MA',
+    'TV-PG', 'TV-Y', 'TV-Y7', 'TV-Y7-FV', 'UR'
+]
+
+
+def convert_to_string(d):
```

Comment:
@rushabh-v I am not quite understanding the purpose of this function as it is not used anywhere?